packages/client/src/hooks/useSuccessMessage.js

```diff
@@ -0,0 +1,41 @@
+import { useModalContext } from "contexts";
+import { TransactionSuccessModal } from "modals";
+import { ASSET_TYPES, ACTION_TYPES } from "../constants/enums";
+
+export default function useSuccessMessage() {
+    const { openModal, closeModal } = useModalContext();
+
+    const showSuccessTransactionModal = (action, safeName, safeAddress) => {
+        const actionData = action.data.actionView;
+        openModal(
+            <TransactionSuccessModal
+                safeName={safeName}
+                safeAddress={safeAddress}
+                actionData={actionData}
+                actionType={getActionType(actionData.type)}
+                txID={action.transactionId}
+                onClose={closeSuccessModal}
+            />,
+            {
+                headerTitle: "Success",
+            }
+        );
+    }
+
+    const getActionType = (type) => {
+        switch (type) {
+            case ACTION_TYPES.TRANSFER_NFT:
+                return ASSET_TYPES.NFT;
```

Comment:
why do we want to map action_type to asset_type? can we just use `action_type` directly?

Reply:
Removed